crates/sui-core/src/authority/authority_store_tables.rs

```diff
@@ -58,6 +58,7 @@ pub struct AuthorityPerpetualTables {
     pub(crate) executed_effects: DBMap<TransactionDigest, SignedTransactionEffects>,
 
     pub(crate) effects: DBMap<TransactionEffectsDigest, TransactionEffects>,
```

Comment:
Could you rename this to synced_effects?

Reply:
Lets do that in a follow up. Right now state sync will write to this table, as well as when effects are written to the executed_effects table so `synced_effects` 100% accurate atm either.

Reply:
It seems either we can remove the write to `effects` table after execution, or we can create another table just for synced effects.